README.rst

```diff
@@ -24,6 +24,10 @@ consult https://tulip-control.sourceforge.io/doc/install.html
 The next section describes how to build documentation.
 A test suite is provided under ``tests/``.  Consult the section "Testing" below.
 
+Pip can install the latest development version too::
+
+  pip install https://github.com/tulip-control/tulip-control/archive/master.zip
```

Comment:
also, this idiom may cause problem with version string identification. I obtain the following results on GNU/Linux Fedora 26 and Python 3.6.2. The shell prompt is `cero%`.
```sh
cero% virtualenv -p python3 f00f
Running virtualenv with interpreter /usr/bin/python3
Using base prefix '/usr'
New python executable in /home/scott/tmp/f00f/bin/python3
Also creating executable in /home/scott/tmp/f00f/bin/python
Installing setuptools, pip, wheel...source done.
cero% source f00f/bin/activate
(f00f) cero% pip install https://github.com/tulip-control/tulip-control/archive/master.zip
Collecting https://github.com/tulip-control/tulip-control/archive/master.zip
  Downloading https://github.com/tulip-control/tulip-control/archive/master.zip
     \ 593kB 7.9MB/s
Collecting networkx<=1.10,>=1.8 (from tulip==1.4.0)
Collecting numpy>=1.7 (from tulip==1.4.0)
  Using cached numpy-1.13.1-cp36-cp36m-manylinux1_x86_64.whl
Collecting omega<0.1.0,>=0.0.9 (from tulip==1.4.0)
Collecting ply>=3.4 (from tulip==1.4.0)
Collecting polytope>=0.2.0 (from tulip==1.4.0)
Collecting pydot>=1.2.0 (from tulip==1.4.0)
Collecting scipy (from tulip==1.4.0)
  Using cached scipy-0.19.1-cp36-cp36m-manylinux1_x86_64.whl
Collecting decorator>=3.4.0 (from networkx<=1.10,>=1.8->tulip==1.4.0)
  Using cached decorator-4.1.2-py2.py3-none-any.whl
Collecting dd==0.4.3 (from omega<0.1.0,>=0.0.9->tulip==1.4.0)
Collecting natsort>=3.5.3 (from omega<0.1.0,>=0.0.9->tulip==1.4.0)
  Using cached natsort-5.1.0-py2.py3-none-any.whl
Collecting pyparsing>=2.1.4 (from pydot>=1.2.0->tulip==1.4.0)
  Using cached pyparsing-2.2.0-py2.py3-none-any.whl
Collecting psutil>=3.2.2 (from dd==0.4.3->omega<0.1.0,>=0.0.9->tulip==1.4.0)
Collecting astutils>=0.0.1 (from dd==0.4.3->omega<0.1.0,>=0.0.9->tulip==1.4.0)
Requirement already satisfied: setuptools>=19.6 in ./f00f/lib/python3.6/site-packages (from dd==0.4.3->omega<0.1.0,>=0.0.9->tulip==1.4.0)
Installing collected packages: decorator, networkx, numpy, ply, pyparsing, pydot, psutil, astutils, dd, natsort, omega, scipy, polytope, tulip
  Running setup.py install for tulip ... done
Successfully installed astutils-0.0.2 dd-0.4.3 decorator-4.1.2 natsort-5.1.0 networkx-1.10 numpy-1.13.1 omega-0.0.11 ply-3.10 polytope-0.2.0 psutil-5.2.2 pydot-1.2.3 pyparsing-2.2.0 scipy-0.19.1 tulip-1.4.0
(f00f) cero% python -c 'import tulip; print(tulip.__version__)'
1.4.0
```